cylc/flow/scripts/cat_log.py

```diff
@@ -264,11 +248,6 @@ def get_option_parser() -> COP:
         help="Job submit number (default=%s, i.e. latest)." % NN,
         metavar="INT", action="store", dest="submit_num", default=NN)
 
-    parser.add_option(
```

Comment:
Not worth keeping and plugging in to a deprecation warning?

Reply:
Unlike the config, I don't think that's needed. We've changed a lot of commands and command options without deprecations. It's pretty easy to change your command line on the fly, if you invoke an obsolete option.